packages/kbn-optimizer/limits.yml

```diff
@@ -144,7 +146,7 @@ pageLoadAssetSize:
   searchQueryRules: 19708
   searchSynonyms: 20262
   security: 81771
-  securitySolution: 98429
+  securitySolution: 159812
```

Comment:
As [you commented](https://github.com/elastic/kibana/pull/223936#issuecomment-2997557995) and in our discussions offline there's some things ([docs](https://docs.elastic.dev/kibana-dev-docs/key-concepts/performance/case-studies/top-level-imports)) we can do to help here. Okay to address in a follow-up for me so long as ops is 👍 with this temporary increase as well.